sharing-service/services/sharing/confirmInvite.js

```diff
@@ -0,0 +1,47 @@
+'use strict'
```

Comment:
The naming of this module (`confirmInvite`) doesn't seem to match the naming conventions for other Lambda handlers and services. We can discuss if this isn't clear.